src/tests/repr.rs

```diff
@@ -28,7 +28,7 @@ fn random_muln_tests<R: PrimeFieldRepr>() {
     for _ in 0..100 {
         let r = R::rand(&mut rng);
 
-        for shift in 0..(r.num_bits()+1) {
+        for shift in 0..(r.num_bits() + 1) {
```

Comment:
Hmm. For short expressions this is fine; for longer expressions with nesting etc., I'd probably bristle at this modifying my spacing choices.